src/main/java/org/example/examplefinalProject/controller/restController/TeacherRestController.java

```diff
@@ -33,5 +32,17 @@ public ResponseEntity<Teacher> detail(@PathVariable int teacherId) {
             return ResponseEntity.notFound().build();
         }
     }
+
+ /*   @PostMapping("/createTeacher")
+    public ResponseEntity<TeacherRecord> createTeacher(@RequestBody TeacherRecord teacherRecord) {
+        teacherService.createTeacher(teacherRecord.teacherName(), teacherRecord.teacherSurname(), teacherRecord.classRoom());
+        return new ResponseEntity<>(teacherRecord, HttpStatus.CREATED);
+    }*/
```

Comment:
why this is commented

Reply:
No longer needed since we put it into TeacherController.